scripts/types.js

```diff
@@ -0,0 +1,101 @@
+/* @flow */
+
+export type Action = {
+  type: string
+};
```

Comment:
Further work here could involve https://flowtype.org/docs/disjoint-unions.html but I don't think it's necessary for this PR